atd-etl/cris_import/cris_import.py

```diff
@@ -517,6 +502,88 @@ def align_db_typing(map_state):
     return map_state
 
 
+def convert_to_ldm_lookup_ids(state):
```

Comment:
I think this function could use a bit of doc about what it is doing. I could see the context of what is happening here getting away from us over time. This is tricky and heady stuff!

Reply:
I agree, totally. I stuck some line-by-line comments of what is going on, and put this out in front of the function:

```
# The following function is used to change, in place, the foreign key values
# from those used by CRIS to those used by the LDM.  This function gets the "import"
# schema name in the argument, and then will use some hard-coded maps between columns 
# and tables to build a SQL command which updates all the appropriate foreign key values
# in the import schema's tables.
```

Reply:
awesome - i think that is very helpful. 🙏